com.github.johnfactotum.Foliate.json

```diff
@@ -0,0 +1,32 @@
+{
+    "app-id": "com.github.johnfactotum.Foliate",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.johnfactotum.Foliate",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [
+        {
+            "name": "foliate",
+            "builddir": true,
```

Comment:
```suggestion
```
It's already set by default

Reply:
Thanks. Wasn't aware of this. I just sort of blindly copied it from Gnome Builder's template.